src/cases/components/index.ts

```diff
@@ -1,7 +1,10 @@
+import { AllocateRoleComponent } from './allocate-role/allocate-role.component';
 import { AlertComponent } from './case-alert/alert.component';
 
 export const components: any[] = [
-  AlertComponent
+  AlertComponent,
+  AllocateRoleComponent
 ];
 
+export * from './allocate-role/allocate-role.component';
 export * from './case-alert/alert.component';
```

Comment:
would this best to move to src\work-allocation-2\components, if this is only used in work allocation 2.